actions/chats.js

```diff
@@ -0,0 +1,76 @@
+import * as types from '../constants/actionTypes';
+import { baseUrl } from '../constants/api';
+import {
+    handleResponse,
+    storeData,
+    addAuthHeader
+} from '../helpers/api';
+
+export const setCurrentChat = (chat) => {
+    return {
+        type: types.SET_CURRENT_CHAT,
+        payload: chat
+    };
+};
+
+const getChats = (data) => {
+    return {
+        type: types.GET_CHATS,
+        payload: data
+    };
+};
+
+export const createMessage = (id, cid, message, user, emojis) => {
+    return {
+        type: types.CREATE_MESSAGE,
+        payload: {
+            id,
+            chats_id: cid,
+            message,
+            user,
+            emojis
+        }
+    };
+};
+
+const createChat = (data) => {
+    return {
+        type: types.CREATE_CHAT,
+        payload: data
+    };
+};
+
+export const fetchChats = (data) => {
```

Comment:
things that need to call the api need to call a function to fetch the data first, before dispatching to redux